### PR TITLE
Added Species model to the API and removed donation columns

### PR DESCRIPTION
**Pull request by jadeharr**
This PR:
* Adds Species to the API
* Removed the donation column from the home and about pages

### CODE REVIEW

File: apps/api/serializers.py [hunk 1]
```diff
@@ -49,23 +71,22 @@ class Meta:
...
-            'people',
-            'planets',
-            # 'url',
-            # 'created',
-            # 'edited'
-        )
\ No newline at end of file
+        model = Species
+
+        fields = {
```
Comment (cunzhig) [inline apps/api/serializers.py:80]: Is there any reason that you uncommented some of the fields?

**Review by cunzhig — COMMENTED** [review]
Minor comment on serializers.py

File: apps/api/serializers.py (hunk 1 above)
Comment (jadeharr) [reply apps/api/serializers.py:80]: I was just going off of what was done with people, I thought the fields that are uncommented are the most interesting. should I uncomment them?

Comment (cunzhig) [reply apps/api/serializers.py:80]: The fields you listed will display in the api page, so I think you can uncomment them.

Comment (jadeharr) [reply apps/api/serializers.py:80]: Okay! Will do.

**Review by cunzhig — APPROVED** [review]

**Pull request merged**